crates/autopilot/src/decoded_settlement.rs

```diff
@@ -128,15 +134,10 @@ impl TryFrom<database::orders::FullOrder> for Order {
             kind: order_kind_from(order.kind),
             sell_token: H160(order.sell_token.0),
             buy_token: H160(order.buy_token.0),
-            sell_amount: big_decimal_to_u256(&order.sell_amount).context("sell_amount")?,
+            sell_amount: big_decimal_to_u256(&order.sell_amount_before_fees)
+                .context("sell_amount")?,
```

Comment:
Why changed to `sell_amount_before_fees`? Because you renamed it in the database query? And then we again save it to `sell_amount`?

Reply:
I think I got confused while writing the custom DB query. The `sell_amount` doesn't get any special treatment so calling it `sell_amount_before_fees` just causes confusion. I think what I actually wanted to do is mention that the `executed_amount` doesn't contain the fees (in case of sell orders).